x-pack/filebeat/module/checkpoint/_meta/docs.asciidoc

```diff
@@ -0,0 +1,158 @@
+[role="xpack"]
+
+:modulename: checkpoint
+:has-dashboards: false
+
+
+== Checkpoint module
+beta[]
+
+This is a module for Checkpoint firewall logs. It supports logs from the Log Exporter in the Syslog format.
```

Comment:
I think we should add a link to the CEF module and mention that it can support receiving data from CheckPoint in CEF format. If there's a trade-off that would be good to explain.

Likewise link from CEF to Checkpoint.